docs/versions/1.0.0.md

```diff
@@ -104,32 +109,37 @@ terms, they are printed in *italics*.
 
 * **bag**: a data package conforming to the [BagIt]{:target=_blank} specifications.
 
-* **bag-ID**: a [urn:uuid]{:target=_blank} given to every deposit upon delivery to DANS. The bag-ID of the first version of a dataset is also known as the *sword token*.
+* **bag-ID**: a [urn:uuid]{:target=_blank} given to every deposit upon delivery to DANS. The bag-ID of the first version
+  of a dataset is also known as the *sword token*.
 
 * **(version N) DANS-bag**: a *bag* that also conforms to (version N) of the DANS BagIt Profile specifications.
 
 * **DANS Data Station**: the publishing and archiving service of DANS for datasets
+  
+* **DANS SWORD Service**: the DANS implementation of the [SWORDv2]{:target=_blank} protocol
 
 * **Data-Station-User-Account**: the account of the *depositor* transferring deposits to DANS
 
-* **dataset**: an information package containing data and metadata published in a *DANS Data Station*. 
-  A dataset may have multiple versions.
+* **dataset**: an information package containing data and metadata published in a *DANS Data Station*. A dataset may
+  have multiple versions.
 
-* **deposit**: a package containing data and metadata sent to DANS by the *depositor* to create a new dataset or dataset version.
+* **deposit**: a package containing data and metadata sent to DANS by the *depositor* to create a new dataset or dataset
+  version.
 
 * **depositor**: the agent sending data to DANS for publishing in a *DANS Data Station*. This term corresponds to the
   Producer in the [OAIS Reference Model]{:target=_blank}.
 
-* **deaccessioned version**: a dataset that is present in a *DANS Data Station*, but no longer disseminated. To the external user the
-  dataset version appears to be deleted, only a tombstone remains.
+* **deaccessioned version**: a dataset that is present in a *DANS Data Station*, but no longer disseminated. To the
+  external user the dataset version appears to be deleted, only a tombstone remains.
 
 * **update-deposit**: a *deposit* that is intended to create a new version of an existing *dataset*.
 
 * **version-bag-ID**: a *bag-ID* that references a *version* of a *dataset*. If the *dataset* consists of several
-  *versions*, the DOI resolves to the most recent of those *version*<hidden>s. All versions of one dataset have the same *sword token*.
+  *versions*, the DOI resolves to the most recent of those *version*<hidden>s. All versions of one dataset have the
+  same *sword token*.
 
-* **sword token**: a [urn:uuid]{:target=_blank} given to first version DEPOSIT bags upon submission through the Sword Service. 
-  This UUID is used when depositing an *update-deposit* and to identify all versions of one dataset. 
+* **sword token**: a [urn:uuid]{:target=_blank} assigned to first version bags upon submission through the
```

Comment:
@lindareijnhoudt : not sure I understood your intentions, please review.

Reply:
LGTM